### ISSUE TITLE
Creating subscriptions sends duplicate parameters

### ISSUE THREAD

**Issue by johnnyoshika** [stripe/stripe-dotnet#1076]
Using:
* .Net Core 2
* Windows 10 Pro 
* Visual Studio 2017

When creating a subscription, the parameters are sent twice in the request body. Here's an example HTTP Request:
```
POST https://api.stripe.com/v1/subscriptions HTTP/1.1

coupon=discount-10p&
plan=stripe-daily-4&
source=tok_1Bc5AUCkr3AsGInl9NHtHxpD&
tax_percent=0&
customer=cus_BynumueEYyVFKv&
coupon=discount-10p&
plan=stripe-daily-4&
source=tok_1Bc5AUCkr3AsGInl9NHtHxpD&
tax_percent=0
```
Notice that every parameter (except for customer) is duplicated in the request body (line breaks added for clarity).

I believe the problem is because `this. ApplyAllParameter` is being called twice from here:  https://github.com/stripe/stripe-dotnet/blob/master/src/Stripe.net/Services/Subscriptions/StripeSubscriptionService.cs#L38-L49

After the first time `this. ApplyAllParameter` is called, the `requestString` that's returned from `this. ApplyAllParameter` looks like this (see the watch window in the screenshot):
![image](https://user-images.githubusercontent.com/504505/34317336-479e1aaa-e761-11e7-90b7-bb75961405ba.png)

The second time `this. ApplyAllParameter` is called, the `requestString` that's returned from `this. ApplyAllParameter` looks like this:
![image](https://user-images.githubusercontent.com/504505/34317341-5e4fa688-e761-11e7-8ad0-e47ce6ab72e2.png)

Notice that all of the parameters are doubled the second time.

**Comment by remi-stripe** [comment]
@johnnyoshika Thanks a lot for the detailed report! I was able to quickly reproduce and push a fix in https://github.com/stripe/stripe-dotnet/pull/1077. This should get reviewed and merged after the holidays!

**Comment by johnnyoshika** [comment]
@remi-stripe Wow, that was fast. I'm looking forward to the PR getting merged. Thanks!

**Comment by ob-stripe** [comment]
Fixed in v11.10.0.
